docs/src/examples/pytorch_cifar.rst

```diff
@@ -2,7 +2,9 @@
 Example with pytorch-cifar
 **************************
 
-# Link to database setup if not done
+.. note ::
+
+    If database not setup or Oríon not installed, follow setup instructions: :doc:`../installing` and :doc:`../database`
```

Comment:
:+1: :smile:

Reply:
We could say in the tutorial how to install orion. It's just ```pip install orion.core```. In the note, we would only point out for the database, and also say something like ```You can test the example without setting up a database by using the option `--debug`, but note that all data gathered during an execution will be lost at the end of it.```